content/docs/1.7.0/references/settings.md

```diff
@@ -472,6 +473,12 @@ Number of seconds that Longhorn allows for the completion of replica rebuilding
 
 Setting that allows you to enable the V1 Data Engine.
 
+#### RWX Volume Fast Failover
```

Comment:
I think we should put the work experimental here as we have done in other places. If the user only looks at available settings to tweak without reading further, there will be no indication that this feature is not as well testes as others.

Reply:
Done.

Reply:
Done.